src/main/java/edu/harvard/iq/dataverse/authorization/providers/builtin/BuiltinUser.java

```diff
@@ -43,7 +44,7 @@
 
     @NotBlank(message = "Please enter a username.")
     @Size(min=2, max=60, message ="Username must be between 2 and 60 characters.")
-    @Pattern(regexp = "[a-zA-Z0-9\\_\\-\\.]*", message = "Found an illegal character(s). Valid characters are a-Z, 0-9, '_', '-', and '.'.")
+    @ValidateUserName(message = "Found an illegal character(s). Valid characters are a-Z, 0-9, '_', '-', and '.'.")
```

Comment:
@sekmiller huh, good catch! Although the message(s) defined above in each tag don't seem to be in the bundle..? `DataverseUserPage.java` only uses "user.username.taken" and "user.username.invalid" -- which are respectively _"This username is already taken."_ and _"This username contains an invalid character."_ -- shouldn't the bundle at least be updated even if the bean validation tags are left with the strings above?